### ISSUE TITLE
how dialated convolution actually work ? 

### ISSUE THREAD

**Issue by SlowMonk** [ibab/tensorflow-wavenet#382]
can someone tell me how this result came out? 
and if change to dialated to 2 ? what will happen??

conv = layers.Conv1D(1, 3, padding='causal',
                     dilation_rate=1,
                     bias_initializer=tf.keras.initializers.zeros)

np.squeeze(sequence.numpy())
->array([0.        , 0.46517092, 0.82355802, 0.99289061, 0.93429699])

np.squeeze(conv.trainable_variables[0].numpy())
->array([-0.19279779,  0.55129746, -0.52658171])

output = conv(sequence)
np.squeeze(output.numpy())
->array([ 0.        , -0.2449505 , -0.17722305, -0.15849652, -0.1033858 ])
